DDDEastAnglia.Tests/Sponsors/sponsor_list/Sponsors_who_havent_paid_yet.cs

```diff
@@ -0,0 +1,31 @@
+using System.Linq;
+using NUnit.Framework;
+
+namespace DDDEastAnglia.Tests.Sponsors.sponsor_list
+{
+    [TestFixture]
+    public sealed class Sponsors_who_havent_paid_yet : context
+    {
+        public Sponsors_who_havent_paid_yet()
```

Comment:
the behaviour in nunit is different.. a constructor acts like the `[TestFixtureSetup]` but I think in xunit the constructor acts more like `[Setup]`

I don't mind.. if it adds more clarity then I'm all for it..